### PR TITLE
Make the subscriber queue size mutable.

### PR DESCRIPTION
**Pull request by kojiterada**
Make the subscriber queue size mutable. #248

### CODE REVIEW

File: tf2_ros/src/tf2_ros/transform_listener.py [hunk 1]
```diff
@@ -47,9 +47,9 @@ def __init__(self, buffer):
             :param buffer: The buffer to propagate changes to when tf info updates.
         """
         self.buffer = buffer
-        self.tf_sub = rospy.Subscriber("/tf", TFMessage, self.callback)
+        self.tf_sub = rospy.Subscriber("/tf", TFMessage, self.callback, queue_size=queue_size)
         self.tf_static_sub = rospy.Subscriber("/tf_static", TFMessage, self.static_callback)
```
Comment (tfoote) [inline tf2_ros/src/tf2_ros/transform_listener.py]: The queue_size should also propogate here.

File: tf2_ros/src/tf2_ros/transform_listener.py [hunk 2]
```diff
@@ -47,9 +47,9 @@ def __init__(self, buffer):
             :param buffer: The buffer to propagate changes to when tf info updates.
```
Comment (tfoote) [inline tf2_ros/src/tf2_ros/transform_listener.py:47]: Please update the doc block too.

**Review by tfoote — CHANGES_REQUESTED** [review]
Thanks this looks like a good extension. I had a few comments. 

And when updating the doc block please also comment about side effecting on other subscribers: http://docs.ros.org/kinetic/api/rospy/html/rospy.topics.Subscriber-class.html wrt to the queue size.

File: tf2_ros/src/tf2_ros/transform_listener.py (hunk 2 above)
Comment (kojiterada) [reply tf2_ros/src/tf2_ros/transform_listener.py:47]: I updated doc block.

File: tf2_ros/src/tf2_ros/transform_listener.py (hunk 1 above)
Comment (kojiterada) [reply tf2_ros/src/tf2_ros/transform_listener.py]: I also change there.

**Review by tfoote — APPROVED** [review]
Thanks for the iteration. And exposing the extra options looks good too.

**Pull request merged**